README.md

```diff
@@ -83,12 +83,15 @@ await runner.setup(__dirname);
 
 Optionally, you can provide "setup" files if you want to copy additional files into the target directory, say from _node_modules_ or a fixtures folder.  You can provide these files as an array of objects.
 
-* `source`: path of the file to copy
-* `destination`: path of where to copy the file to
+- `source`: path of the file to copy (can be a string or URL object)
+- `destination`: path of where to copy the file to (can be a string or URL object)
 
 A third options object can be provided with the following supported options
+
 - `create` - automatically create the directory provided in the first param (default is `true`)
 
+Using string paths:
```

Comment:
I would say let's make this and the other line h4's in markdown, e.g.
- `#### Using string path`
- `#### Using URL objects`